py/vtdb/vtgate_utils.py

```diff
@@ -75,3 +77,71 @@ def wrapper(self, *args, **kwargs):
           delay = min(max_delay_ms, delay)
     return wrapper
   return decorator
+
+class VitessError(Exception):
+  """VitessError is raised by an RPC  with a server-side application error.
```

Comment:
Should this be a child of Exception or a DatabaseError? Also, fix "RPC  with" double space.